doc/oemof_outputlib.rst

```diff
@@ -11,26 +11,116 @@ as the requirements to plotting functions greatly depend on individial requireme
+
+See the `pandas documentation <http://pandas.pydata.org/pandas-docs/stable/>`_  to learn how to `visualise <http://pandas.pydata.org/pandas-docs/version/0.18.1/visualization.html>`_, `read or write <http://pandas.pydata.org/pandas-docs/stable/io.html>`_ or how to `access parts of the DataFrame <http://pandas.pydata.org/pandas-docs/stable/advanced.html>`_ to process them.
+
+The documentation of the outputlib consists of three parts:
+
 .. contents::
     :depth: 1
     :local:
     :backlinks: top
 
-The main purpose of the outputlib is to collect and organise results.
-It gives back the results as a python dictionary holding pandas Series for scalar values and pandas DataFrames for all nodes and flows between them. This way we can make use of the full power of the pandas package available to process the results. 
+The first step is the processing of the results (:ref:`oplref1`).
+This is followed by basic examples of the general analysis of the results
+(:ref:`oplref2`) and finally the use of functionalities already included in the
+outputlib for providing a quick access to your results (:ref:`oplref3`).
+Especially for larger energy systems the general approach will help you to
+write your own results processing functions.
 
-See the `pandas documentation <http://pandas.pydata.org/pandas-docs/stable/>`_  to learn how to `visualise <http://pandas.pydata.org/pandas-docs/version/0.18.1/visualization.html>`_, `read or write <http://pandas.pydata.org/pandas-docs/stable/io.html>`_ or how to `access parts of the DataFrame <http://pandas.pydata.org/pandas-docs/stable/advanced.html>`_ to process them.
+.. _oplref1:
 
 Collecting results
 ------------------
 
 Collecting results can be done with the help of the processing module:
 
 .. code-block:: python
-    
+
     results = outputlib.processing.results(om)
 
-The scalars and sequences describe nodes (with keys like (node, None)) and flows between nodes (with keys like (node_1, node_2)). You can directly extract the data in the dictionary by using these keys, where "node" is the name of the object you want to address. If you want to address objects by their label, you can convert the results dictionary such that the keys are changed to strings given by the labels:
+The scalars and sequences describe nodes (with keys like (node, None)) and
+flows between nodes (with keys like (node_1, node_2)). You can directly extract
+the data in the dictionary by using these keys, where "node" is the name of
+the object you want to address.
+Processing the results is the prerequisite for the examples in the following
+sections.
+
+.. _oplref2:
+
+General approach
+----------------
+As stated above after processing you will get a dictionary with all result
+data. At the first try, the keys of the dictionary might seem to have rather
+complex structure, but very useful as well.
```

Comment:
I am not sure what is meant by this exactly ;-) That the structure is complex but useful?

Reply:
yeah, you are right. I removed the sentence.